src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.tsx

```diff
@@ -86,6 +82,16 @@ type Props = {
    */
   selection: GlobalSelection;
 
+  /**
+   * Custom default selection values (e.g. a different default period)
+   */
+  defaultSelection?: Partial<GlobalSelection>;
```

Comment:
Used by Performance.

Reply:
I think this warrants a bigger discussion with Product /cc @adhiraj?

Reply:
@markstory What's your thinking? My gut is that most people are ok with (and perhaps prefer) 7d to 14d given sentry's natural usage frequency is weekly. I can do a quick pulse on the team channel too.
Also just to confirm, the default is used by issues, releases, and user feedback?

Reply:
@adhiraj The default is used everywhere except Performance/Discover - so all the above mentioned + Events/Discover (which is going away soon) and Dashboards